Vialer/VoIPGRIDRequestOperationManager.m

```diff
@@ -94,25 +93,12 @@ - (void)loginWithUser:(NSString *)user password:(NSString *)password success:(vo
             [[NSNotificationCenter defaultCenter] postNotificationName:LOGIN_SUCCEEDED_NOTIFICATION object:nil];
 
             // Fetch SIP account credentials
-            NSString *appAccountUrl = [responseObject objectForKey:@"app_account"];
-            if ([appAccountUrl isKindOfClass:[NSString class]]) {
-                [self retrievePhoneAccountForUrl:appAccountUrl success:^(AFHTTPRequestOperation *operation, id responseObject) {
-                    NSObject *account = [responseObject objectForKey:@"account_id"];
-                    NSObject *password = [responseObject objectForKey:@"password"];
-                    if ([account isKindOfClass:[NSNumber class]] && [password isKindOfClass:[NSString class]]) {
-                        [self setSipAccount:[(NSNumber *)account stringValue] andSipPassword:(NSString *)password];
-                    } else {
-                        [self setSipAccount:nil andSipPassword:nil];
-                    }
-                    success(operation, success);
-                } failure:^(AFHTTPRequestOperation *operation, NSError *error) {
-                    [self setSipAccount:nil andSipPassword:nil];
-                    success(operation, success);
-                }];
-            } else {
-                [self setSipAccount:nil andSipPassword:nil];
-                success(operation, success);
-            }
+            [self updateSIPAccountWithSuccess:^{
+                if (success) success(operation, success);
```

Comment:
I have an issue with the use of the ```operation``` variable here. We currently return use four(!) variables in this function which are all called ```operation```! (two times on line 68, twice on line 72). I would suggest some renaming there!